Package.swift

```diff
@@ -86,6 +94,11 @@ let package = Package(
                  * only.  Unconditionally define it instead.
                  */
                 .define("WIN32_LEAN_AND_MEAN"),
+                /*
+                 * These defines are required on WASM, to disable use of pthread.
```

Comment:
How will this play with web assembly threads? Do we have some way to stay up to speed with whether those are supported in Swift Wasm/WASI? cc @MaxDesiatov

Reply:
WASI-libc has some rudimentary support for pthread API, but we shouldn't assume it's available, I'm not sure SwiftWasm has that enabled. Even when it is, with very high probability it may just use noop stubs under the hood. Very likely we may need a separate triple altogether for multi-threaded Wasm.